scripts/make-release

```diff
@@ -1,30 +1,41 @@
 #!/usr/bin/env bash
 
-source "$(dirname "$0")/common.sh"
```

Comment:
Perhaps rename this file to `release.sh`?

Reply:
Jenkins will bark at the filename change; I am almost certain it hardcodes script/make-patch-release somewher

Reply:
I've updated the Jenkinsfile to fix this.  I've thought about moving everything from `common.sh` to `make-release` but decided against it for now, as make-release neatly contains only the top-level code.